openedx/features/course_experience/views/course_home.py

```diff
@@ -248,3 +264,26 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         }
         html = render_to_string('course_experience/course-home-fragment.html', context)
         return Fragment(html)
+
+
+def _get_next_up_banner_fragment(assignment_title, resume_course_url):
+    if assignment_title and resume_course_url:
+        return _generate_up_next_banner_html(assignment_title, resume_course_url)
+    return None
+
+
+def _generate_up_next_banner_html(assignment_title, resume_course_url, assignment_duration='10 min'):
```

Comment:
`assignment_duration` is never passed in yet, right?

Reply:
Correct, we don't have this information yet on how long a sequential will take and that seemed out of scope for this PR when the AC is 
```
Copy: Get started on what's next: [Unit Name] (x min)

Unit name is a clickable link

Estimate total time, or use 10 minutes as a proxy for now.
```